integration-cli/cli/cli.go

```diff
@@ -222,3 +223,14 @@ func WithStdin(stdin io.Reader) func(*icmd.Cmd) func() {
 		return nil
 	}
 }
+
+// GenerateRandomAlphaOnlyString generates an alphabetical random string with length n.
+func GenerateRandomAlphaOnlyString(n int) string {
```

Comment:
Perhaps we should move it somewhere inside docker/internal/test/, together with the existing unit tests; that marks the package as "internal" only

Reply:
Oh, looks like we have a "testutil" directory there already; https://github.com/moby/moby/blob/master/internal/testutil/

Reply:
OK

Reply:
You can create a file there, and move the `GenerateRandomAlphaOnlyString()` function there, including the associated tests from https://github.com/moby/moby/blob/master/pkg/stringutils/stringutils_test.go

As I noticed in https://github.com/moby/moby/pull/35034#pullrequestreview-68582683, `GenerateRandomASCIIString()` is not used anywhere, so that can be removed from the `pkg/stringutils/` as well (I suggest doing that all in a single PR/commit so that it's easier to track where things went)

Reply:
Yes. As I comment it above, it will be done next. And would plz check PR [func Ellipsis() at docker/cli](https://github.com/docker/cli/pull/621)

Reply:
Oh, missed that comment; perfect!

Let me review that PR, I see you updated 👍